libproxy/cmake/modules/pacrunner_webkit.cmk

```diff
@@ -6,7 +6,7 @@ if(WIN32)
     link_directories(WEBKIT_LIBRARIES)
     include_directories("${WEBKIT_INCLUDE_DIR}")
   endif()
-elseif(APPLE)
+elseif(APPLE AND WITH_APPLE_WEBKIT)
```

Comment:
Probably needs to be split - as otherwise, the else statement is valid on apple systems with 'disabled webkit'

Reply:
Yes, I also find the nested if as in your original patch much cleaner. It will prevent the need for modification of the `pkgconfig.cmk`.

Reply:
I was thinking we might eventually want to build a version of `libproxy` on `macOS` with `WITH_APPLE_WEBKIT=OFF` (telling it not to use the Apple provide JavaScriptCore) and `WITH_WEBKIT3=ON`.   We would want this to pick up WebKitGTK the same way it does on non macOS systems.  Is that a valid thing to want to do?